### ISSUE TITLE
Blink escape character not working

### ISSUE THREAD

**Issue by rashil2000** [mintty/mintty#1097]
So I try running in bash (MSYS2)

```bash
echo  -e "\033[5mBlinking Text\033[0m"
```

But the text does not blink. Do I have to do any setting? My .minttyrc is this:

```
Transparency=off
Font=RecMonoSemicasual NF
FontHeight=11
FontWeight=400
BoldAsFont=yes
BoldAsColour=no
Scrollbar=right
ConfirmExit=no
ThemeFile=google-light
Term=xterm-256color
BellType=0
TabBar=9
SessionGeomSync=4
SessionCommands=Kali Linux:-i D:\Data\Pictures\Icons\kali-menu.ico wsl ~;Arch Linux:-i D:\Data\Pictures\Icons\archlinux.ico wsl -d arch-linux ~;Ubuntu - Azure:-i D:\Data\Pictures\Icons\ubuntuazure.ico ssh ubuntuazure2;Ubuntu - OSBoxes:-i D:\Data\Pictures\Icons\Ubuntu.ico ssh ubuntuosboxes;
Columns=90
Rows=30
```

Tested with both `MSYS=enable_pcon` and `MSYS=disable_pcon`

**Comment by Biswa96** [comment]
Add this option `AllowBlinking=yes` in minttyrc file.

**Comment by rashil2000** [comment]
Thanks for the test file!

I couldn't see any option for Blinking in the `Options` dialog. Does it need to be manually done in minttyrc?

**Comment by mintty** [comment]
"Allow blinking" used to be in the Options dialog but it got lost by default due to some revision, without specific reason. Maybe I should add it back in.
The default value is false because blinking is considered an accessibility issue as some people have physical reactions to blinking.

**Comment by rashil2000** [comment]
> The default value is false because blinking is considered an accessibility issue as some people have physical reactions to blinking.

Yes yes, that's totally understandable :)

**Comment by BrianInglis** [comment]
It could be useful if developers would use it sparingly with bold inverse coloured messages *ONLY* when highlighting form fields with no or invalid values to be supplied or corrected. 
I often have to scroll forms up and down a couple of times to spot the location of an issue. 
But it was overused as well as being an irritating unaesthetic nightmare in most designs.

**Comment by Biswa96** [comment]
> some people have physical reactions to blinking.

What! Now I have to learn human biology before creating a program 😮

**Comment by mintty** [comment]
Released 3.5.0.
